cmd/piped/BUILD.bazel

```diff
@@ -18,9 +19,52 @@ go_binary(
     visibility = ["//visibility:public"],
 )
 
+# Create a passwd file with a root and pipecd user and uid.
+passwd_entry(
+    name = "root_user",
+    gid = 0,
+    uid = 0,
+    username = "root",
+    home = "/root",
+    info = "Root",
+)
+
+passwd_entry(
+    name = "pipecd_user",
+    gid = 1000,
+    uid = 1000,
+    username = "pipecd",
+    home = "/home/pipecd"
+    info = "PipeCD User",
```

Comment:
nit: "pipecd"

Reply:
Is the uppercase letter not preferable?

Reply:
Yes, I think we should use only the lowercase to reduce the unknown risks.

Reply:
Gotcha!